src/GlobalStyle.js

```diff
@@ -2,7 +2,20 @@ import React from 'react';
 import { createGlobalStyle } from 'styled-components';
 
 const StyledGlobalSyle = createGlobalStyle`
-  margin:1em;
+  
+  *{
+
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  } 
+
+body {
+  margin: 0;
```

Comment:
Maybe you can order your attributes by a specific order like alphabetical oder by logic.

Reply:
good idea